draft-ietf-ppm-dap.md

```diff
@@ -2411,6 +2430,37 @@ to the end as needed to produce the final aggregate output.
 Collectors must also maintain state for the lifetime of each task, which
 includes key material associated with the HPKE key configuration.
 
+## Aggregation Jobs & Replay Protection
```

Comment:
Note that we never use "&" in place of "and" in this draft.
```suggestion
## Aggregation Jobs and Replay Protection
```

Reply:
aagh, sorry, I keep making that mistake